cms/sass/pages/_journal-details.scss

```diff
@@ -22,16 +22,16 @@
 
       @media (min-width: 769px) {
         padding-bottom: 0;
-        width: calc(100% / 3 - 20px);
+        width: calc(100% / 3 - $spacing-03a);
```

Comment:
Here the `min-width` is still absolute, and since this is a width calculation, is a relative value correct?  This is not a text element so does the same rule apply?